### ISSUE TITLE
ffmpeg ENOENT, no live video, no way to use locally installed ffmpeg nor replace malfunctioning.version 

### ISSUE THREAD

**Issue by genuinefafa** [colinbendell/homebridge-blink-for-home#152]
Refs #11 

I'm trying to connect a
Blink Mini firmware 9.9

With homebridge docker image running on Raspberry Pi 4
Ubuntu Jammy Jellyfish (22.04.3 LTS)
Node.js Version: v18.19.0

[homebridge-blink-for-home](https://github.com/colinbendell/homebridge-blink-for-home) 3.8.2
```
# uname -a
Linux LibreELEC-living 5.10.110 #1 SMP Sat Jan 7 12:15:32 UTC 2023 aarch64 aarch64 aarch64 GNU/Linux
```

I think there is something wrong with ffmpeg, but I'm not entirelly sure what can it be.

After installation there were some errors while trying to run ffmpeg; 

```
[1/28/2024, 1:21:12 PM] [Blink] Blink G8V1-9000-1306-67X8 - Failed to start ffmpeg: spawn /homebridge/node_modules/homebridge-blink-for-home/node_modules/ffmpeg-for-homebridge/ffmpeg ENOENT
```

I did:

a) check if the included ffmpeg works, manually, via terminal access and it says `bash: ./ffmpeg: No such file or directory` which is strange since the file is there and it has X permission on all security levels
a) use locally installed ffmpeg (that works with other plugins, for other camera vendors) vi a symbolic link
b) download latest ffmpeg-for-homebridge for different architecture (armv7-32, armv8-64), overwriting `homebridge-blink-for-home/node_modules/ffmpeg-for-homebridge` executable file

when using an executable version of ffmpeg, I can see - in the logs - that there is "something" going on (at least, running, no ENOENT error).

```
[1/27/2024, 9:19:35 PM] [Blink] Blink G8V1-9000-1306-67X8 - ffmpeg -hide_banner -loglevel warning -loop 1 -framerate 1 -re -f image2 -i /homebridge/node_modules/homebridge-blink-for-home/src/homebridge/../offline.png -c:v libx264 -preset ultrafast -pix_fmt yuv420p -profile:v baseline -s 1280x720 -g 300 -r 10 -an -sn -dn -b:v 299k -bufsize 598k -maxrate 299k -payload_type 99 -f rtp -ssrc 13197736 -srtp_out_suite AES_CM_128_HMAC_SHA1_80 -srtp_out_params ********* srtp://192.168.1.11:49733?rtcpport=49733&localrtcpport=49733&pkt_size=1378
[1/27/2024, 9:19:36 PM] [Blink] Blink G8V1-xxxx - STDOUT: SDP:
v=0
o=- 0 0 IN IP4 127.0.0.1
s=No Name
c=IN IP4 192.168.1.11
t=0 0
a=tool:libavformat 60.3.100
m=video 49733 RTP/AVP 99
b=AS:299
a=rtpmap:99 H264/90000
a=fmtp:99 packetization-mode=1
a=crypto:1 AES_CM_128_HMAC_SHA1_80 inline:*********
```

but there is no video on any HomeKit client, therefore, no live video (i'm seeing "offline.png")
the snapshot/thumb does work, doh.

**In other words**
 * Confirm there are several versions of ffmpeg installed and running on container
 * Confirm that there is something wrong with included ffmpeg (at least in my env)
 * Can create snapshots
 * Cannot see live video

any ideas?
